common/eth2/src/lighthouse/attestation_rewards.rs

```diff
@@ -0,0 +1,37 @@
+use serde::{Deserialize, Serialize};
+
+// Details about the rewards paid for attestations
+// All rewards in GWei
+
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
+pub struct IdealAttestationRewards {
+    // Validator's effective balance in gwei
+    pub effective_balance: u64,
```

Comment:
We should quote each of these using

```rust
#[serde(with = "eth2_serde_utils::quoted_u64")]
```

Leave it off the `i64`s, we don't have anything for them yet (https://github.com/sigp/ethereum_serde_utils/issues/2).